### PR TITLE
#163412741  Connect red-flag routes to database

### PR DESCRIPTION
**Pull request by Manorlds-Eaglespark**
Please leave a comment, thanks.

@JaymesKat @matthewacha @alexxsanya @jennizalwango @Wabuluka @noahkalyesubula @lizz24 @kalsmic @k7ko @bekeplar @e-ian @masete @Manorlds-Eaglespark @nanfuka @HarrietAkot @PatrickMugayaJoel @kyakusahmed @Anguandia @CandySusan @kabaliisa @bisonlou @HabibSentongo @v1b3m @kdenno @davicken @dtnnml @Araali1 @natalie-abbie @ernesthenry @JamesMudidi

### CODE REVIEW

File: .coverage [hunk 1]
```diff
@@ -1 +1 @@
...
...s\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\databases\\database.py":[1,2,3,4,7,9,11,19,45,54,60,66,72,81,88,95,102,109,116,122,13,14,15,16,17,20,31,41,42,43,61,62,63,64,46,49,50,51,52,123,124,125,126,55,57,58],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\tests\\test_redflags.py":[1,2,3,4,5,7,8,26,31,37,52,59,77,84,93,103,113,123,135,156,164,184,193,215,222,232,244,255,9,10,11,12,13,15,16,18,19,20,21],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\auth\\__init__.py":[1,3,5],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\auth\\views.py":[2,3,4,5,6,7,8,9,10,11,12,13,14,17,18,20,64,65,67,117,118,120,121,122,123,125,126,127,128,22,24,25,26,27,28,29,30,32,33,35,36,39,40,45,46,47,48,49,50,51,52,53,55,56,57,58,69,71,72,74,75,77,79,81,82,83,85,86,87,88,89,90,92,93,94,96,97,99,113,114,104,105,101,102,41,42,60,61],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\utilities\\register_validation.py":[1,5,6,7,8,9,10,11,15,17,26,18,19,20,21,22,23,24,27,29,31,33,35,37,39,41,30,32,40,38,36,28,34],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\utilities\\login_validation.py":[4,5,9,10,14,11,12,15,17,19,21,18,20,16]}}
```
Comment (JaymesKat) [inline .coverage:1]: This should not be included in your github repo

File: app/databases/database.py [hunk 2]
```diff
@@ -54,8 +54,12 @@ def save_user(self, user):
     def create_default_admin(self):
         postgres_insert_user_query = ("INSERT INTO users ("
                                       "firstname, lastname, othernames, email, password,"
-                                      "phonenumber, username, isadmin, registered) VALUES (  'Christine','Turky','Sweeri','christinet@gmail.com','{0}','013234565','Sweeri','True', '{1}') ON CONFLICT (email) DO NOTHING;".format(Bcrypt().generate_password_hash("asdfdsaf").decode(), datetime.now()))
-        self.cursor.execute(postgres_insert_user_query)
+                                      "phonenumber, username, isadmin, registered) SELECT 'Christine','Turky','Sweeri',%s,%s,'013234565','Sweeri','True', %s WHERE NOT EXISTS (SELECT id FROM users WHERE email= %s );")
+        password = Bcrypt().generate_password_hash("asdfdsaf").decode()
+        time_registered = datetime.now()
+        email = 'christinet@gmail.com'
```
Comment (JaymesKat) [inline app/databases/database.py:60]: why is this email hard-coded here?

File: tests/.coverage [hunk 3]
```diff
@@ -1 +1 @@
...3,134,54,55,56,57,58,59,116,117,118,119,120,41,42,43,44,45,99,100,101,102,103,104,105,107,108,109,110,111,112,113,26,27,28,29,30,31,32,34,35,36,37,38,138,139,140,141,142,143,144,146,147,148,149,150,151,152,63,64,65,66,67,68,69,71,72,73,74,75,76,77,156,157,158,159,160,161,162,164,165,166,167,168,169,170,81,82,83,84,85,86,87,89,90,91,92,93,94,95,175,176,177,178,179,180,181,183,184,185,186,187,188,189,193,194,195,196,197,198,200,201,202,203,204,205,206],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\auth\\__init__.py":[1,3,5],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\auth\\views.py":[2,3,4,5,6,7,8,9,10,11,12,13,16,17,19,60,61,63,106,107,109,110,111,112,114,115,116,117,65,67,68,70,71,73,75,76,77,78,79,81,82,83,85,86,88,102,103,93,94,90,91,21,23,24,25,26,27,28,29,31,32,34,36,37,38,39,42,43,44,45,46,47,48,49,50,52,53,54,56,57],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\utilities\\register_validation.py":[1,5,6,7,8,9,10,11,15,17,26,18,19,20,21,22,23,24,27,29,31,33,35,37,39,41,30,32,40,38,36,28,34],"C:\\Users\\mk\\Desktop\\Andela_Bootcamp\\i_reporter_api\\app\\utilities\\login_validation.py":[4,5,9,10,14,11,12,15,17,19,21,18,20,16]}}
```
Comment (JaymesKat) [inline tests/.coverage:1]: exclude the `.coverage` file

File: .coverage (hunk 1 above)
Comment (Manorlds-Eaglespark) [reply .coverage:1]: Added to .gitignore, thanks.

File: app/databases/database.py (hunk 2 above)
Comment (Manorlds-Eaglespark) [reply app/databases/database.py:60]: Removed them, thanks.

File: tests/.coverage (hunk 3 above)
Comment (Manorlds-Eaglespark) [reply tests/.coverage:1]: Done, thanks

**Pull request merged**